docs/advanced_reference/commands/autonomy_mint.md

```diff
@@ -171,6 +189,12 @@ autonomy mint service [OPTIONS] PACKAGE_PATH
 `--threshold`
 : Threshold for the minimum number of agents required to run the service. The threshold has to be at least $\lceil(2N + 1) / 3\rceil$, where $N$ is total number of the agents in the service.
 
+`--token ERC20_TOKEN_ADDRESS`
+: Use the ERC20 token for bonding.
+
+`--update TOKEN_ID`
+: Update the existing mint with the latest package hash.
```

Comment:
same suggestion of previous comment

Reply:
Addressed [c241c59](https://github.com/valory-xyz/open-autonomy/pull/2050/commits/c241c59d71cdbcfc957398e1e6eb69eaef5daab1)